services/federator/src/Federator/ExternalServer.hs

```diff
@@ -66,13 +84,27 @@ callLocal = runInwardError . callLocal'
     toResponse (Left err) = InwardResponseError err
     toResponse (Right bs) = InwardResponseBody bs
 
-callLocal' :: (Members '[Service, Embed IO, TinyLog, Polysemy.Reader RunSettings, Polysemy.Error InwardError] r) => Request -> Sem r ByteString
-callLocal' req@Request {..} = do
+callLocal' ::
+  ( Members
+      '[ Service,
+         Embed IO,
+         TinyLog,
+         DiscoverFederator,
+         Polysemy.Reader RunSettings,
+         Polysemy.Error InwardError
+       ]
+      r
+  ) =>
+  Maybe ByteString ->
+  Request ->
+  Sem r ByteString
+callLocal' mcert req@Request {..} = do
   Log.debug $
     Log.msg ("Inward Request" :: ByteString)
       . Log.field "request" (show req)
+      . maybe id (\cert -> Log.field "certificate" (Text.decodeUtf8 cert)) mcert
```

Comment:
I forgot, but should this be logging only for now, or should there be some other actions to run? The PR is still in the draft state so perhaps that explains it.

Reply:
We removed the logging statement now.